neogradle/docs/nonmclibs.md

```diff
@@ -0,0 +1,53 @@
+# Non-Minecraft dependencies
+
+Non-Minecraft dependencies are not loaded by NeoForge by default in the development environment. To get NeoForge to recognize the non-Minecraft dependency, they must be added to runs as a runtime dependency.  
+
+For example, you can add the `com.example:example` library to all runs like so:
+```gradle
+runs {
+    configureEach {
+        dependencies {
+            // highlight-next-line
+            runtime 'com.example:example:1.0'
+        }
```

Comment:
Does this also need to be added to the main dependencies block? (Compile time or runtime)